code/modules/arcade/mob_hunt/mob_datums.dm

```diff
@@ -139,10 +139,6 @@
 	for(var/A in area_blacklist)
 		for(var/areapath in typesof(A))
 			possible_areas[areapath] -= 2
-	//removes "bad areas" which shouldn't be on-station but are subtypes of station areas. probably should the unused ones and consider repathing the rest
-	var/list/bad_areas = list(subtypesof(/area/construction), /area/solar/derelict_starboard, /area/solar/derelict_aft)
-	for(var/A in bad_areas)
-		possible_areas -= A
```

Comment:
I need to check in on this, I removed this a LONG time ago when I started this project

Reply:
works like it used to from what I can tell